src/main/scala/chiseltest/package.scala

```diff
@@ -167,4 +174,11 @@ package object chiseltest {
   implicit def decoupledToDriver[T <: Data](x: ReadyValidIO[T]) = new DecoupledDriver(x)
 
   implicit def validToDriver[T <: Data](x: ValidIO[T]) = new ValidDriver(x)
+
+  def intervalToBigDecimal(i: Interval): BigDecimal = {
```

Comment:
I still don't think this belongs in the base API. Perhaps as experimental or internal, but there should be no expectation of continued support here since this is going into chisel base?